26-Vertical Slider/script.js

```diff
@@ -0,0 +1,33 @@
+const slideLeft = document.querySelector('.left-slide')
+const slideRight = document.querySelector('.right-slide')
+const upButton = document.querySelector('.up-button')
+const downButton = document.querySelector('.down-button')
+const slidesLength = slideRight.querySelectorAll('div').length
+
+let activeSlideIndex = 0
+
+slideLeft.style.top = `-${(slidesLength - 1) * 100}vh`
+
+upButton.addEventListener('click', () => changeSlide('up'))
```

Comment:
Can you think of a way to imbed the direction into the html element so that we don't have to specify the `direction` in lines 11 and 12. 
ex. 
```
[upBotton, downButton].forEach((button) => button.addEventListener('click', changeSlide));
```

then the `changeSlide` function could end up being looking something like:
```
const changeSlide = (evt) => {
 const { target } = evt; 
 const direction =  target.{FIGURE_OUT_THIS};
 ...
 ...
}
```

there are many ways to do this, one of them is the `data attribute` way, google/Youtube it if you need hints.